code/modules/client/preferences.dm

```diff
@@ -1566,6 +1566,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 	return 1
 
 /datum/preferences/proc/copy_to(mob/living/carbon/human/character)
+	character.change_species(species) // Yell at me if this causes everything to melt

```

Comment:
This comes across as something you mayyyy not want to do, but I"m not sure---I know there's other cases where this is used and the species is explicitly set, later on....soooo not sure, TBH.